### PR TITLE
Add aodh, logging and metric-storage services

### PR DESCRIPTION
**Pull request by vyzigold**
This PR adds aodh, logging and metric-storage to the list of services to gather their configuration secrets.

### CODE REVIEW

**Comment by vyzigold** [comment]
I also added *.monitoring.rhobs to the gathered CRs, which are CRs created by telemetry-operator for metric storage

File: collection-scripts/gather_crs [hunk 1]
```diff
@@ -10,9 +10,11 @@ fi
 # Resource list
 crs=()
 
-# explictly adding rabbit in the grep because it's not part of the openstack
-# umbrella and we rely on the upstream rabbitmq-cluster-operator
-for i in $(/usr/bin/oc get crd | grep -E "(openstack|rabbitmq)\.(org|com)" | awk '{print $1}')
+# Explictly adding rabbit in the grep because it's not part of the openstack
```
Comment (dprince) [inline collection-scripts/gather_crs:13]: the rabbitmq operator is actually part of the umbrella in the sense that we: install it, and create the rabbit cluster as part of OpenStackControlplane. I guess you just mean the CRD is named with a different domain?

Comment (vyzigold) [reply collection-scripts/gather_crs:13]: I actually didn't write the sentence about rabbitmq, I just made the "E" a capital letter and added a "." to make it into a sentence. I'm not sure what exactly the original author meant. The change, which comes with this PR is the second sentence about monitoring.rhobs.

**Review by fmount — CHANGES_REQUESTED** [review]

File: collection-scripts/gather_crs (hunk 1 above)
Comment (vyzigold) [reply collection-scripts/gather_crs]: I changed the message as requested

**Review by fmount — APPROVED** [review]
/lgtm

**Pull request merged**